### ISSUE TITLE
No module named 'surrealism' after installing

### ISSUE THREAD

**Issue by simonvanderveldt** [Morrolan/surrealism#14]
Good evening Ian,

Really liking the package! I wanted to try it as described in the readme but ran into the following:

```
root@80b2fb250389:/# pip install surrealism
Collecting surrealism
  Downloading surrealism-0.13.2.tar.gz (118kB)
    100% |████████████████████████████████| 122kB 1.6MB/s
Requirement already satisfied (use --upgrade to upgrade): setuptools in /usr/local/lib/python3.5/site-packages (from surrealism)
Installing collected packages: surrealism
  Running setup.py install for surrealism ... done
Successfully installed surrealism-0.13.2
root@80b2fb250389:/# python
Python 3.5.2 (default, Jul 19 2016, 21:09:41)
[GCC 4.9.2] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import surrealism
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named 'surrealism'
```

Any idea what could cause this?
The above was with Python 3.5, but I get the same with 2.7.

**Comment by Morrolan** [comment]
Hi, many thanks for letting me know about this!  I have no idea what happened there, but I've pushed a new, source-only version (I think the python eggs were to blame).  The new version is 0.13.3.  Please can you try again?

Also, not sure if you know but `pip install surrealism` will only install the package for your default Python version.  However pip has multiple versions, so you can ensure you get the right version using `pip3.5 install surrealism` etc.

**Comment by simonvanderveldt** [comment]
> Hi, many thanks for letting me know about this! I have no idea what happened there, but I've pushed a new, source-only version (I think the python eggs were to blame). The new version is 0.13.3. Please can you try again?

Thanks for the quick response! Just tried the new version and unfortunately it doesn't seem to work for me.
I'm not 100% sure I'm correct but it seems like it only installs the egg, the actual package isn't installed.

```
(.venv) root@933e7035dce0:/# pip install surrealism
Collecting surrealism
  Downloading surrealism-0.13.3.tar.gz (118kB)
    100% |████████████████████████████████| 122kB 1.5MB/s
Requirement already satisfied (use --upgrade to upgrade): setuptools in /.venv/lib/python3.5/site-packages (from surrealism)
Installing collected packages: surrealism
  Running setup.py install for surrealism ... done
Successfully installed surrealism-0.13.3
You are using pip version 8.1.1, however version 8.1.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
(.venv) root@933e7035dce0:/# ls -al .venv/lib/python3.5/site-packages/
total 40
drwxr-xr-x  9 root root 4096 Jul 26 14:41 .
drwxr-xr-x  3 root root 4096 Jul 26 14:41 ..
drwxr-xr-x  2 root root 4096 Jul 26 14:41 __pycache__
-rw-r--r--  1 root root  126 Jul 26 14:41 easy_install.py
drwxr-xr-x 11 root root 4096 Jul 26 14:41 pip
drwxr-xr-x  2 root root 4096 Jul 26 14:41 pip-8.1.1.dist-info
drwxr-xr-x  5 root root 4096 Jul 26 14:41 pkg_resources
drwxr-xr-x  5 root root 4096 Jul 26 14:41 setuptools
drwxr-xr-x  2 root root 4096 Jul 26 14:41 setuptools-20.10.1.dist-info
drwxr-xr-x  2 root root 4096 Jul 26 14:41 surrealism-0.13.3-py3.5.egg-info
```

> Also, not sure if you know but pip install surrealism will only install the package for your default Python version. However pip has multiple versions, so you can ensure you get the right version using pip3.5 install surrealism etc.

Thanks, I know. I run everything in containers, so no problems there :)

**Comment by Morrolan** [comment]
OK no worries, leave it with me as I can now replicate this in an Ubuntu VM.

**Comment by Morrolan** [comment]
Fixed in 0.14.9.
